spec/unit/etc_spec.rb

```diff
@@ -0,0 +1,155 @@
+require 'puppet'
+require 'spec_helper'
+
+# The Ruby::Etc module is largely non-functional on Windows - many methods
+# simply return nil regardless of input, the Etc::Group struct is not defined,
+# and Etc::Passwd is missing fields
+describe Puppet::Etc, :if => !Puppet.features.microsoft_windows? do
+  let(:bin) { 'bin'.force_encoding(Encoding::ISO_8859_1) }
+  let(:root) { 'root'.force_encoding(Encoding::ISO_8859_1) }
```

Comment:
Would it make sense to vary the encodings here to be something other than `Encoding::ISO_8859_1` for each property?

Also - should we bring in our standard multibyte UTF-8 here somewhere?  I'm not sure I see anything validating that UTF-8 still works as desired.